graylog2-server/src/main/java/org/graylog/plugins/views/search/views/ViewDTO.java

```diff
@@ -48,7 +50,9 @@
 @AutoValue
 @JsonDeserialize(builder = ViewDTO.Builder.class)
 @WithBeanGetter
+@DbEntity(collection = "dashboards", readPermission = RestPermissions.DASHBOARDS_READ)
```

Comment:
Does this prevent us from adding an entity for saved searches? Maybe not a problem, but if we plan to add that any time soon, we should think about how that would work.

Reply:
Yes, I was also thinking about this. I took this as a chance to find out if it works and just implemented it in 927e2ec35ba2e14aa8effdfb561326ebc84d1517 because it was trivial.

Reply:
Moved this to a separate PR: https://github.com/Graylog2/graylog2-server/pull/19761